docs/lib/storage/fragments/js/copy.md

```diff
@@ -0,0 +1,116 @@
+## Copy
+
+`Storage.copy` method copies an existing file to a different key.
+
+It returns a `{key: new S3 Object key}` object on success:
+
+<amplify-callout>
+Storage.copy can copy an object up to 5 GB in a single operation.
+</amplify-callout
+
+### Options
+
+```typescript
+await Storage.copy(
+	src: { 
+		key: string,
+		level?: 'private' | 'protected' | 'public',
+		identityId?: string, // only works if you are copying from a protected file 
+	},
+	dest: { 
+		key: string,
+		level?: 'private' | 'protected' | 'public',
+	}, 
+	config?: {
+		track?: boolean, // if true, automatically send Storage Events to Amazon Pinpoint
+	}
+)
+```
+
+### Other Options
+```typescript
+await Storage.copy({ key: 'src' }, { key: 'dest' }, {
+	acl?: string, // https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html#upload-property
+	cacheControl?: string, // Specifies caching behavior along the request/reply chain
+	expires?: Date, // The date at which the object is no longer cacheable
+	metadata?: [key: string]: string, // A map of metadata to store with the object in S3
+})
+```
+
+### Copy file
+
+For the most basic use case, you can copy a file from the specified key to another key within the same [File Access
+Level](~/lib/storage/configureaccess.md) (Defaults to 'public').
```

Comment:
sift up to the line above. I don't think our docs will interpret this correctly.